lc56.py

```diff
@@ -1 +1,24 @@
+class Solution:
+    def merge(self, intervals: List[List[int]]) -> List[List[int]]:
+        def isOverlap(a,b):
+            maxStart = max(a[0],b[0])
+            minFinish = min(a[1],b[1])
+            return (minFinish >=  maxStart) # overlap
+        
+        intervals.sort(key = lambda x:x[0])
+        res = []
+        temp = intervals[0]
+        res.append(temp)
+        
+        for i in range(1,len(intervals)):
+            if isOverlap(temp,intervals[i]):
```

Comment:
I don't think there is a need for this function. You can use: `if res[-1][1] >= intervals[i][0]:`